daemon/daemon_main.go

```diff
@@ -1410,7 +1411,16 @@ func (d *Daemon) instantiateAPI() *restapi.CiliumAPI {
 }
 
 func initKubeProxyFreeOptions() {
-	if option.Config.EnableNodePort &&
+	if option.Config.EnableKubeProxyFree {
+		option.Config.EnableNodePort = true
```

Comment:
Should we also include an override of`option.Config.EnableExternalIPs` to true here?